setup.py

```diff
@@ -22,6 +22,7 @@
     packages=find_packages(),
     include_package_data=True,
     license='MIT License',
+    install_requires=["django>=1.7"],
```

Comment:
This has caused me problems in the past. See https://github.com/charettes/django-admin-enhancer/issues/23

I think it's best to treat Django minimum version as a documentation issue rather than something for setup.py to handle